core/src/main/java/org/elasticsearch/indices/IndicesService.java

```diff
@@ -130,7 +130,7 @@ protected void doStop() {
         }
         try {
             if (latch.await(shardsClosedTimeout.seconds(), TimeUnit.SECONDS) == false) {
-              logger.warn("Not all shards are closed yet, waited {}sec - stopping service", shardsClosedTimeout.seconds());
+                logger.warn("Not all shards are closed yet, waited {}sec - stopping service", null, shardsClosedTimeout.seconds());
```

Comment:
If `shardsClosedTimeout` is a `TimeValue` then its `toString` should output a sane suffix. Forcing this into seconds doesn't seem required.